shared/js/background/experiments.es6.js

```diff
@@ -0,0 +1,95 @@
+const settings = require('./settings.es6')
+const atbUtils = require('./atb-utils.es6')
+const retentionExperiments = require('../../data/experiments-out')
+const ATB_FORMAT_RE = /(v\d+-\d(?:[a-z_]{2})?)$/
+class Experiment {
+    constructor () {
+        this.variant = ''
+        this.atbVariant = ''
+        this.activeExperiment = {}
+    }
+
+    getVariant () {
+        const atbVal = settings.getSetting('atb')
+        if (atbVal && atbVal.match(ATB_FORMAT_RE) && atbVal[atbVal.length - 2].match(/[a-z]/i)) {
+            this.variant = atbVal[atbVal.length - 2]
+        } else {
+            this.variant = '_'
+        }
+        return this.variant
+    }
+
+    getATBVariant () {
+        const atbVal = settings.getSetting('atb')
+        if (atbVal && atbVal.match(ATB_FORMAT_RE) && atbVal[atbVal.length - 1].match(/[a-z]/i)) {
+            this.atbVariant = atbVal[atbVal.length - 1]
+        } else {
+            this.atbVariant = '_'
+        }
+        return this.atbVariant
+    }
+
+    setActiveExperiment () {
+        settings.ready()
+            // TODO: REMOVE THIS
+            // .then(settings.updateSetting('atb', 'v212-5rz'))
```

Comment:
🚨🚨🚨
@jdorweiler uncomment this to force a variant that is opted into the experiment. This will enable the banner.

The logging below will help with testing.

I'll remove all this before merge.